### PR TITLE
Changes to Enemy attack

### PR DESCRIPTION
**Pull request by Sand3r-**
Resoloves #16 
- added cooldown to enemies damage
- damage formulae is now randomized
- damage dealt is now shown on Zion's Bible

### CODE REVIEW

File: Zion's End/src/Grid.cpp [hunk 1]
```diff
@@ -149,6 +149,21 @@ void Grid::Render(sf::RenderTarget& renderer)
 	}
 }
 
+#pragma warning(push)
+#pragma warning(disable: 4715)
+sf::Vector2f Grid::GetBiblePosition()
```
Comment (Daroslaw) [inline Zion's End/src/Grid.cpp:154]: So what about handling the situation of not finding an end tile?

**Review by Daroslaw — CHANGES_REQUESTED** [review]
Only one thing that imho needs some additional care, overall good job!

File: Zion's End/src/Grid.cpp (hunk 1 above)
Comment (Sand3r-) [reply Zion's End/src/Grid.cpp:154]: There are two way out of this:
- Assume that we are the only authors of the maps, and that we aren't trying to crash it on purpose
- Create a map validator, that would check for Starting and End Tile presence.
- Return {0, 0}

Comment (Daroslaw) [reply Zion's End/src/Grid.cpp:154]: You forgot to change word "two" to "three" on comment edit 😛 
Something like `return {-1, -1} `would do in our case.
Just to soothe the compiler's conscience

Comment (Sand3r-) [reply Zion's End/src/Grid.cpp:154]: Hm, yeah right :smile: 
It's coming then

**Review by Daroslaw — APPROVED** [review]

**Pull request merged**